plugins/carbon_black_defense/unit_tests/util.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import sys
+import os
+import json
+
+from komand_carbon_black_defense.connection.connection import Connection
+from komand_carbon_black_defense.actions.find_event.schema import Input
+from komand_carbon_black_defense.connection.schema import Input
+
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+
+class Util:
+    @staticmethod
+    def read_file_to_dict(filename):
+        with open(filename, "rt") as my_file:
+            return json.loads(
+                Util.read_file_to_string(os.path.join(os.path.dirname(os.path.realpath(__file__)), filename))
+            )
```

Comment:
I feel like this file has many good utilities in it. I think we should consider adding some of these to a shared library we can use across plugins

Reply:
Nice idea! Would like to help with that if that's something the team agrees to. Is there something I can do with this PR to kick that off or no? like copying into a separate file that other plugins can import?